### PR TITLE
Lobby setup

### LINKED ISSUE #8: Greeting Banner
A user should see a "Welcome" greeting and the current date at the top of the Lobby

### LINKED ISSUE #11: Feeling Prompt
A user should see a heading prompting them to choose from a list of 4 emotions

### LINKED ISSUE #12: Feeling Buttons
A user should see 4 different emotions presented as buttons on the lobby page

- [x] Sad
- [x] Angry
- [x] Anxious
- [x] Fine

### LINKED ISSUE #13: Action Prompt
A user should see a prompt for what page they would like to visit after establishing their current mood.

"What do you want to do about it?"

### LINKED ISSUE #14: Navigation
A user should see 3 "action" buttons that lead to different "rooms" in the app.

- [x] Reflect (journaling room)
- [x] Feel (Sound stimulation room)
- [x] Breathe (breathing animation room)

These buttons do not need to be routed at this point in the process

### LINKED ISSUE #15: Logo
A user should see a logo on the lobby page. This logo will remain on every page for navigation back to the lobby.

### LINKED ISSUE #16: Quote of the day Scroller
A user should see a motivational quote of the day at the bottom of the lobby page.

- This quote will later be populated using an api, but a placeholder can be used for the time being

### PR DESCRIPTION
**Pull request by Kristiannmiller**
## Description
This PR 
- Creates the basic framework of the Lobby/Home page.
- Adds basic styling to elements on the Lobby/Home page.
- Adds footer element to the App component
- Adds basic styling to footer element of the App component
- Changes the 'root' element to the body tag in index.html
- closes #8 
- closes #11 
- closes #12 
- closes #13 
- closes #14 
- closes #15 
- closes #16 

## Motivation and Context
This PR sets up our Lobby/Home page to start working with React Router.

## How Has This Been Tested?
- [X] Other Please specify: No tests yet, but testing ideas have been added to corresponding test files (App.scss & Lobby.scss)

## Types of changes
- [X] New feature (non-breaking change which adds functionality)
- [X] Styling / Design

## Checklist:
- [X] My code follows the code style of this project.
- [X] My change requires a change to the documentation.
- [X] I have added test NOTES to cover my changes.

### CODE REVIEW

File: src/Components/Lobby/Lobby.scss [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  align-self: center;
+  text-align: center;
+  justify-content: center;
+  height: 80vh;
+  width: 80vw;
+}
+.moodButton {
+  border-radius: 100px;
+  margin:1vh;
+  width: 15vh;
```
Comment (nathanielmillard) [inline src/Components/Lobby/Lobby.scss:13]: Nice way of using vh on both height and width to keep a 1:1 apsect ration on these buttons!

**Review by nathanielmillard — APPROVED** [review]
Over all I have to say this looks good, its a great start on meeting the wireframe without getting to bogged down in styling just yet. 
I am excited to get working on testing, and actually hooking up the interactive elements to functionality!

File: src/Components/Lobby/Lobby.test.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import { render, screen } from '@testing-library/react';
+import Lobby from './Lobby';
+
+test('renders learn react link', () => {
+  render(<Lobby />);
+  const linkElement = screen.getByText(/learn react/i);
+  expect(linkElement).toBeInTheDocument();
+});
+
+// It should render a welcome greeting
```
Comment (BrigetteDoelp) [inline src/Components/Lobby/Lobby.test.js:10]: LOVE the pseudo code start for the testing file. Even though we aren't writing the actual tests just yet, it's great that we're actively keeping testing in the front of our minds as we go along.

Comment (nathanielmillard) [reply src/Components/Lobby/Lobby.test.js:10]: I agree, it's important to keep testing in mind while we are writing.

**Review by BrigetteDoelp — APPROVED** [review]

**Pull request merged**